requirements.txt

```diff
@@ -0,0 +1,13 @@
+bumpversion==0.5.3
+coverage==4.4.2
+pre-commit>=2.12.0,<2.13
+mock==2.0.0
+packaging==16.8
+pytest-runner>=3.0.0,<3.1.0
+pytest>=3.2.0,<3.3.0
+readme-renderer>=24.0
+setuptools>=40.6.2
+git+git://github.com/sceptre/sceptre.git@master#egg=sceptre
```

Comment:
Why install Sceptre's master? Why not just say `sceptre>=2.7`?

Reply:
I'm thinking that it might be better to not pin here just to make it less of a maintenance burden.

Reply:
That's not a pin, it's just a version requirement. It wouldn't be appropriate for hooks or resolvers, but template handlers simply won't work prior to version 2.7.0. So anything greater than 2.7.0 is appropriate in the case of template handlers.

Reply:
sorry, i misread your previous comment.  I have change to require sceptre from pypi.